azure-pipelines.yml

```diff
@@ -952,12 +952,6 @@ jobs:
     matrix:
       ${{ each arch in parameters.architectures }}:
         # https://wiki.ubuntu.com/Releases
-        Bionic ${{ arch.name }}:
-          # 18.04
-          deb_system: ubuntu
-          distro: bionic
-          arch: ${{ arch.value }}
-          pool: ${{ arch.pool }}
```

Comment:
I am surprised that the `TestDebPackages` job for `Bionic` has been dropped as early as #15833.